### PR TITLE
OLS-801: Create introductory content

### PR DESCRIPTION
**Pull request by rh-tokeefe**
CP to TP1 branch

Version(s): CP to TP branch

Issue: https://issues.redhat.com/browse/OLS-801

Link to docs preview:
https://78606--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/about/ols-about-openshift-lightspeed.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@JoaoFula please provide QA review. Thanks!

**Comment by JoaoFula** [comment]
Hi @rh-tokeefe , Mostly good. I do have a question about the section **about data**.
It appears as a subchapter of **Large Language Model (LLM) Requirements** in the index on the right. 
However, both in the **Large Language Model (LLM) Requirements** chapter organization, which excludes the **about data**, and from it's content, it feels like it should be it's own chapter.
That being said, not entirely sure about the title **about data** as indeed it suggest a continuation of the rest of the **about <LLM>** that's shown previously. Maybe **data usage**? Not sure if this naming is required by legal or something though

**Comment by rh-tokeefe** [comment]
@JoaoFula Thanks for the suggestion. I agree and will definitely move the topic "up" one level in the hierarchy. 

It's part of the house style to use "About" in the title of conceptual topics, so I'll go with About data use. Thanks!

**Comment by JoaoFula** [comment]
LGTM

**Comment by thoraxe** [comment]
LGTM

File: modules/ols-openshift-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: CONCEPT
+[id="ols-openshift-requirements"]
+= OpenShift Requirements 
+:context: ols-openshift-requirements
+
+{ols-long} requires {ocp-product-title} 4.15 or later running on x86 hardware. Any installation type or deployment architecture is supported so long as the cluster is 4.15+ and x86-based.
+
+For the {ols-long} {ols-release} release, the cluster you use must be connected to the Internet and it must have telemetry enabled. Telemetry is enabled by default, so please confirm that if you are using a standard installation process for {ocp-short-name} that it does not disable telemetry.
```
Comment (gabriel-rh) [inline modules/ols-openshift-requirements.adoc:8]: For the {ols-long} {ols-release} release, the cluster you use must be connected to the Internet and it must have telemetry enabled. Telemetry is enabled by default, so please confirm that, if you are using a standard installation process for {ocp-short-name}, it does not disable telemetry.    maybe?

**Review by gabriel-rh — COMMENTED** [review]
some small comments

File: modules/ols-about-data-use.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+= About data use
+:context: ols-about-data-use
+
+{ols-full} is a virtual assistant you interact with using natural language. Using the {ols-long} interface, you send chat messages that {ols-long} transforms and sends to the Large Language Model (LLM) provider you have configured for your environment. These messages can contain information about your cluster, cluster resources, or other aspects of your environment.
+
+The {ols-long} {ols-release} release has limited capabilities to filter or redact the information you provide to the LLM. Do not enter information into the {ols-long} interface that you do not want to send to the LLM provider.
+
+By using the {ols-long} as part of the {ols-release} release, you agree that Red Hat may use all of the messages that you exchange with the LLM provider for any purpose. The transcript recording data uses the Red Hat Insights system’s back-end, and is subject to the same access restrictions and other security policies. 
+
+You may contact mailto:openshift-lightspeed-alpha@redhat.com[Red Hat] and request that your data be deleted at the end of the {ols-release} release period.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ols-about-data-use.adoc:12]: Interesting, when I tested this using a local build it worked using Mozilla. It also worked just now using Chrome. Just curious: what browser you use @gabriel-rh ?

File: modules/ols-large-language-model-requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+* OpenAI
+
+* Azure OpenAI
+
+* WatsonX
+
+[NOTE]
+====
+Many self-hosted or self-managed model servers claim API compatibility with OpenAI. It is possible to configure the OpenShift Lightspeed OpenAI provider to point to an API-compatible model server. If the model server is truly API-compatible, especially with respect to authentication, then it may work. These configurations have not been tested by Red Hat, and issues related to their use are outside of the scope of {ols-release} support.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ols-large-language-model-requirements.adoc:16]: done

File: modules/ols-openshift-requirements.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ols-openshift-requirements.adoc:8]: @gabriel-rh The sentence is long so I ended up just breaking it like so:

Telemetry is enabled by default. If you are using a standard installation process for {ocp-short-name} confirm that it does not disable telemetry.

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**